### ISSUE TITLE
relion_helix_inimodel2d

### ISSUE THREAD

**Issue by sprombhu** [3dem/relion#1210]
Hello,

I ran this

relion_helix_inimodel2d --i IniModel/run1_it005.star \
  --iniref 1@IniModel/run1_it005_reconstructed.mrcs  --angpix 1.15 \
  --maxres 6 --search_angle 2 --step_angle 0.5 --mask_diameter 250 \
  --j 6 --iter 5  --o IniModel/run2

and got this error

ERROR:
Projector::computeFourierTransformMap%%ERROR: Dimension of the data array should be 2 or 3.

I went to see possible solutions, and found this

git clone heep://bforsbe@bitbucket.org/scheres/relion-3.0_beta.git

might help but instead I have this

Cloning into 'relion-3.0_beta'...
fatal: Unable to find remote helper for 'heep'.

Would I please get some helps here?

Kind regards,
-S

**Comment by biochem-fan** [comment]
> relion-3.0_beta

This is an extremely old version which we no longer support.
Using this version probably does not solve your problem.

**Comment by sprombhu** [comment]
I am actually working on 4.0. Will the command work the same way?

**Comment by sprombhu** [comment]
Originally was because of this

ERROR:
Projector::computeFourierTransformMap%%ERROR: Dimension of the data array should be 2 or 3

from this

relion_helix_inimodel2d --i IniModel/run1_it005.star
--iniref 1@IniModel/run1_it005_reconstructed.mrcs --angpix 1.15
--maxres 6 --search_angle 2 --step_angle 0.5 --mask_diameter 250
--j 6 --iter 5 --o IniModel/run2.

**Comment by biochem-fan** [comment]
It should be but I am not an expert on helical processing.

Did you read https://pmc.ncbi.nlm.nih.gov/articles/PMC7008511/?

**Comment by biochem-fan** [comment]
I guess you are using `--iniref` in a wrong way.

The paper says:
> Optionally, the user can provide an initial 2D reference image using the --iniref option. Provided that it is rescaled to the same pixel size and re-windowed to the same box size as the input 2D class averages (using the relion_image_handler program), this initial reference could, for example, come from the low-resolution XY structure obtained with the 2D class average spanning an entire cross-over.

Isn't `IniModel/run1_it005_reconstructed.mrcs` a 3D volume?

**Comment by sprombhu** [comment]
That is totally why. Thank you!

**Comment by scheres** [comment]
No, iniref should be a 2D image (we're trying to reconstruct the xy-crosssection. I'm not sure the n@stack.mrcs syntax works though, could you make an individual image first?
relion_image_handler --i  1@IniModel/run1_it005_reconstructed.mrcs  --o iniref.mrc
